### ISSUE TITLE
Broken Image Links for Avatars

### ISSUE THREAD

**Issue by bengott** [TelescopeJS/Telescope#693]
From this twitter comment/conversation asked by @Primigenus (Rahul):
https://twitter.com/Rahul/status/553835276389584896

The problem appears to be that if a user signs up via twitter or another linked service and then changes his/her avatar image on twitter/other service, the user.profile object does not get updated, so the image load fails and initials are shown instead. I'll try to track it down and implement a bugfix soon.

**Comment by bengott** [comment]
I finally looked into this, and my initial assessment seems to be incorrect. I just tried logging in with twitter then changing my twitter avatar. Upon logging out and then logging back into Telescope, the image URL gets updated. If I remember correctly, that functionality is handled by the built-in Meteor user accounts system.

The only situation I can imagine where the image link could get broken is if the user changed their avatar elsewhere and then didn't login to Telescope for a long time and the linked service removed the image?

Can anyone else verify this?
